src/imitation/algorithms/preference_comparisons.py

```diff
@@ -142,7 +142,10 @@ def __init__(
         # will set self.logger, which also sets the logger for the algorithm
         super().__init__(custom_logger)
         if isinstance(reward_fn, reward_nets.RewardNet):
-            reward_fn = reward_fn.predict
```

Comment:
As with `adversarial/common.py` I do not see why this is necessary, think we can just use `predict_processed` always

Reply:
Agreed. It would be good to just simply this one.